13:4: not a valid test operator: (
13:4: not a valid test operator: 470.129.06
  0%|          | 0/5 [00:00<?, ?it/s] 20%|██        | 1/5 [31:58<2:07:55, 1918.87s/it] 40%|████      | 2/5 [1:07:34<1:42:19, 2046.61s/it] 60%|██████    | 3/5 [1:44:23<1:10:40, 2120.45s/it] 80%|████████  | 4/5 [2:22:31<36:26, 2186.88s/it]  prune.py:293: RuntimeWarning: invalid value encountered in double_scalars
  control_weights = 100000 * abs(connectivity[ind]) / max(connectivity) # * prev_weight
100%|██████████| 5/5 [3:02:00<00:00, 2252.38s/it]100%|██████████| 5/5 [3:02:00<00:00, 2184.09s/it]
  0%|          | 0/5 [00:00<?, ?it/s] 20%|██        | 1/5 [31:56<2:07:44, 1916.15s/it] 40%|████      | 2/5 [1:07:31<1:42:16, 2045.37s/it] 60%|██████    | 3/5 [1:44:25<1:10:43, 2121.97s/it] 80%|████████  | 4/5 [2:22:38<36:29, 2189.62s/it]  100%|██████████| 5/5 [3:02:09<00:00, 2255.09s/it]100%|██████████| 5/5 [3:02:09<00:00, 2185.92s/it]
  0%|          | 0/5 [00:00<?, ?it/s]